serverlessworkflow/README.adoc

```diff
@@ -0,0 +1,56 @@
+= Kogito Serverless Workflow documentation
+
+== Prerequisite
+
+Running Antora with Lunr extension (required to generate self-hosted search) requires Node.js v16.0.0 or above.
+
+== Installing Antora and Lunr
+
+Install Antora CLI 3.0.0 within the playbook project so you can run the antora command using npx:
+
+`npm i -D -E @antora/cli@3.0.0`
+
+Next, install the site generator package within the playbook project:
+
+`npm i -D -E @antora/site-generator@3.0.0`
+
+Now when you run `npx antora -v`, you should see the version of both the Antora CLI and the site generator printed in the terminal.
+
+```
+@antora/cli: 3.0.0
+@antora/site-generator: 3.0.0
+```
+
+Finally install the Lunr extension with:
+
+`npm i @antora/lunr-extension`
+
+== Building the website locally
+
+This folder contains an antora playbook, `antora-playbook.yml` to generate the documentation. To run and generate the documentation locally under the `build` folder use:
+
+`npx antora antora-playbook.yml`
```

Comment:
Personally, I would mention the `antora-playbook-author.yml` for local development; otherwise, if someone runs this command, no local changes will be visible on the generated website as Antora will use the content from the blessed repository.

Reply:
Sorry, I missed it earlier. but updated in the new commit.